policy.php

```diff
@@ -0,0 +1,35 @@
+<?php
+	require_once("template.php");
+	require_once("config.php");
+	Template::render_header("Privacy Policy");
+
+	echo "<h1>" . _("Privcacy Policy") . "</h2>";
+	echo "<h2>" . _("Who we are") . "</h2>";
+	echo WHO_WE_ARE;
+	echo "<h2>" . _("Contact") . "</h2>";
+	echo POLICY_NAME . "<br>";
+	echo ADDRESS . "<br>";
+	echo POLICY_MAIL . "<br>";
+	echo POLICY_PHONE . "<br>";
```

Comment:
Maybe check whether empty as it is not required? The additional `<br>` could be quite annoying

Reply:
Only phone and "who we are" are not required. Or do you mean in general?

Reply:
Yeah I mean the phone, if you don't fill it in it will still print the extra br tag